pkg/test/framework/features/features.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package features
+
+// TODO: this file should be generated from YAML to make it more easy to modify.
+
+// WARNING: changes to existing elements in this file will cause corruption of test coverage data.
+// don't change existing entries unless absolutely necessary
+
+type Feature string
+
+const (
+	UsabilityObservabilityStatus              Feature = "Usability.Observability.Status"
```

Comment:
It would be nice to have some explaination of what the existing ones here represent and if I am implementing a new test/feature how I pick the feature string

Reply:
Yes, I'd like to get this generated from yaml, and have a readme with a description of the top level items.  Unfortunately, we don't yet have consensus on what all the top levels will be.  I am working on that, and am happy to hold this review open until we have a better idea.

Reply:
No I don't think we need to block this in it, but we should figure that out before we go about filling all of these out to avoid having to redo things of course